CollapseLauncher/Classes/RepairManagement/Genshin/Fetch.cs

```diff
@@ -454,7 +454,7 @@ private YSDispatchDec InitializeMasterDecryptor()
             YSDispatchDec decryptor = new YSDispatchDec();
 
             // Initialize the master key
-            decryptor.InitMasterKey(ConfigV2.MasterKey, ConfigV2.MasterKeyBitLength, RSAEncryptionPadding.Pkcs1);
+            decryptor.InitMasterKey(LauncherMetadataHelper.CurrentMasterKey.Key, LauncherMetadataHelper.CurrentMasterKey.BitSize, RSAEncryptionPadding.Pkcs1);
```

Comment:
Check that `CurrentMasterKey.Key` is not `null`, this could break a lot of things.